packages/cli/src/commands/promote.ts

```diff
@@ -0,0 +1,123 @@
+import chalk from 'chalk';
+import type Client from '../util/client';
+import getArgs from '../util/get-args';
+import getProjectByCwdOrLink from '../util/projects/get-project-by-cwd-or-link';
+import { getPkgName } from '../util/pkg-name';
+import handleError from '../util/handle-error';
+import { isErrnoException } from '@vercel/error-utils';
+import logo from '../util/output/logo';
+import ms from 'ms';
+import requestPromote from '../util/promote/request-promote';
+import promoteStatus from '../util/promote/status';
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** move utils to command directory

In this or a follow-up PR: ‏Can we colocate these files with the only place that uses them? They should be in `packages/cli/src/commands/promote/*` and the current `promote.ts` could be the `indext.ts` in that directory.

Reply:
Done!